x-pack/elastic-agent/docs/README.md

```diff
@@ -4,3 +4,17 @@ Elastic Agent Documentation
 
 The source files for the Elastic Agent documentation are currently stored
 in the [observability-docs](https://github.com/elastic/observability-docs) repo.
+
+## Testing docker container
```

Comment:
How about us moving some of the instructions/tips to the top level `elastic-agent/README.md`. Or at least have a link telling you there are some instructions?

Reply:
As far as I know, this is the only README we have. Github shows it automatically like the top level one. Happy to move it to the top. It is also the place I was looking for it initially.